includes/class-llms-sessions.php

```diff
@@ -131,14 +132,28 @@ protected function end( $start ) {
 			)
 		);
 
+		if ( ! is_wp_error( $end ) ) {
+			global $wpdb;
+			$wpdb->query( // phpcs:ignore: WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching
```

Comment:
Just learned this cool shorthand implemented by the WPCS project to make ignoring these warnings shorter.

`WordPress.DB.DirectDatabaseQuery.DirectQuery` = `db call ok`
`WordPress.DB.DirectDatabaseQuery.NoCaching` = `no-cache ok`

```suggestion
			$wpdb->query( // db call ok; no-cache ok.
```